chef/data_bags/crowbar/migrate/nova/011_configure_block_device_allocation.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade ta, td, a, d
```

Comment:
Unused method argument - `ta`. If it's necessary, use `_` or `_ta` as an argument name to indicate that it won't be used.<br>Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.<br>Use def with parentheses when there are parameters.